design/cluster-scope-resource-filter.md

```diff
@@ -0,0 +1,572 @@
+
+### Using scenarios:
+#### no namespaced resources + no cluster resources
+This is not allowed. Backup or restore cannot contain no resource.
+
+#### no namespaced resources + some cluster resources
+The following commands all mean backup no namespaced resources and some cluster resources.
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaces=*
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=StorageClass
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=*
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=StorageClass
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaces=*
+	--exclude-namespaced-resources=*
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=StorageClass
+```
+
+#### no namespaced resources + all cluster resources
+The following commands all mean backup no namespaced resources and all cluster resources.
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaces=*
+	--include-cluster-resources=true
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaces=*
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=*
+	--include-cluster-resources=true
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=*
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaces=*
+	--exclude-namespaced-resources=*
+	--include-cluster-resources=true
+```
+
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaces=*
+	--exclude-namespaced-resources=*
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=*
+```
+
+#### some namespaced resources + no cluster resources
+This means backup all resources in namespaces default and kube-system, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--include-cluster-resources=false
+```
+
+This means backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in all namespaces, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--include-cluster-resources=false
+```
+
+This means backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in namespace default and kube-system, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--include-cluster-resources=false
+```
+
+This means backup all resources except Ingress type resources in all namespaces, and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--exclude-namespaced-resources=Ingress
```

Comment:
Also, `--include-namespaced-resources="*"`

Reply:
Alternatively, you could use `--exclude-resourcs=Ingress` without the include param since the old params include everything by default.